mobile/lib/features/trade/domain/trade_values.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:get_10101/features/trade/domain/direction.dart';
+import 'package:get_10101/features/trade/domain/leverage.dart';
+import 'package:get_10101/ffi.dart';
+import 'package:get_10101/common/domain/model.dart';
+
+class TradeValues {
+  Amount margin;
+  double quantity;
+  Leverage leverage;
+  Direction direction;
+
+  double price;
+  double liquidationPrice;
+  Amount fee;
+  double fundingRate;
+
+  TradeValues(
+      {required this.direction,
+      required this.margin,
+      required this.quantity,
+      required this.leverage,
+      required this.price,
+      required this.liquidationPrice,
+      required this.fee,
+      required this.fundingRate});
+
+  factory TradeValues.create(
+      {required double quantity,
+      required Leverage leverage,
+      required double price,
+      required double fundingRate,
+      required Direction direction}) {
+    Amount margin =
+        Amount(api.calculateMargin(price: price, quantity: quantity, leverage: leverage.leverage));
+    double liquidationPrice = api.calculateLiquidationPrice(
+        price: price, leverage: leverage.leverage, position: direction.intoPosition());
+
+    // TODO: Calculate fee based on price, quantity and funding rate
+    Amount fee = Amount(30);
```

Comment:
What fee is this?

Reply:
Note: I just depicted this as it is depicted in Figma.
My assumption is, that it is the fee for the first 24h.

In ItchySats we defined two fees when opening a position:

1. The first 24h of "rollovers" (can be a positive or negative "fee")
2. An optional opening fee charged by the maker (always a positive value on the taker side); that we never used in production

Given that we have an orderbook now, I am not sure what this fee should represent; I suppose it could be the fee for the "orderbook service" as well. This might need some more thought!

Reply:
Created a ticket for that: https://github.com/get10101/10101/issues/139